packages/ui-components/style/styling.css

```
@@ -86,3 +88,16 @@ select.jp-mod-styled {
   -webkit-appearance: none;
   -moz-appearance: none;
 }
+
+select.jp-mod-styled:not([multiple]) {
+  height: 32;
```

I guess the unit is `px`

```suggestion
  height: 32px;
```